content/influxdb/cloud-iox/query-data/flux-sql.md

```diff
@@ -0,0 +1,522 @@
+which is fundamentally different from IOx.
+Because of this, Flux is less performant when querying an IOx-powered bucket.
+However, as a full-featured scripting language, Flux gives you the flexibility
+to perform a wide range of data processing operations such as statistical
+analysis, alerting, HTTP API interactions, and other operations that aren't
+supported in SQL.
+By using Flux and SQL together, you can benefit from both the performance of SQL
+and the flexibility of Flux.
+
+## What to do in SQL versus Flux?
+
+We recommend doing as much of your query as possible in SQL for the most
+performant queries.
+Do any further processing in Flux.
+The following chain of Flux functions can be replicated in SQL:
+
+```js
+from() |> range() |> filter() |> aggregateWindow()
+```
+
+#### Example Flux versus SQL queries
+
+{{< expand-wrapper >}}
+{{% expand "View example basic queries" %}}
+
+{{% influxdb/custom-timestamps %}}
+##### Flux
+```js
+from(bucket: "get-started")
+    |> range(start: 2022-01-01T08:00:00Z, stop: 2022-01-01T20:00:00Z)
+    |> filter(fn: (r) => r._measurement == "home")
+```
+
+##### SQL
+```sql
+SELECT *
+FROM home
+WHERE time >= '2022-01-01T08:00:00Z' AND time < '2022-01-01T20:00:00Z'
+```
+{{% /influxdb/custom-timestamps %}}
+
+_For more information about performing basic queries with SQL, see
+[Perform a basic SQL query](/influxdb/cloud-iox/query-data/sql/basic-query/)._
+
+{{% /expand %}}
+
+{{% expand "View example aggregate queries" %}}
+
+{{% influxdb/custom-timestamps %}}
+##### Flux
+```js
+from(bucket: "get-started")
+    |> range(start: 2022-01-01T08:00:00Z, stop: 2022-01-01T20:00:00Z)
+    |> filter(fn: (r) => r._measurement == "home")
+    |> filter(fn: (r) => r._field == "temp" or r._field == "hum")
+    |> aggregateWindow(every: 2h, fn: mean)
+```
+
+##### SQL
+```sql
+SELECT
+  DATE_BIN(INTERVAL '2 hours', time, '1970-01-01T00:00:00Z'::TIMESTAMP) AS _time,
+  room,
+  avg(temp) AS temp,
+  avg(hum) AS hum,
+FROM home
+WHERE
+  time >= '2022-01-01T08:00:00Z'
+  AND time < '2022-01-01T20:00:00Z'
+GROUP BY room, _time
+ORDER BY _time
+```
+{{% /influxdb/custom-timestamps %}}
+
+_For more information about performing aggregate queries with SQL, see
+[Aggregate data with SQL](/influxdb/cloud-iox/query-data/sql/aggregate-select/)._
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+## Use SQL and Flux together
+
+To use SQL and Flux together and benefit from the strengths of both query languages,
+build a **Flux query** that uses the [`iox.sql()` function](/flux/v0.x/stdlib/experimental/iox/sql/)
+to execute a SQL function.
+The SQL query should return the base data set for your query.
+If this data needs further processing that can't be done in SQL, those operations
+can be done with native Flux.
+
+{{% note %}}
+The process below uses the `/api/v2/query` endpoint and can be used to execute
+SQL queries against an InfluxDB IOx-powered bucket with all existing
+InfluxDB 2.x clients.
```

Comment:
We could mention here infuxdb data source for grafana cloud so those users don't feel like they can't currently integrate (since the flight-sql plugin is install only)

Also worth mentioning users could just use a raw rest request here as you mentioned with `/api/v2/query`

Reply:
Good call. I added the following list:

- InfluxDB 2.x client libraries
- Grafana and Grafana Cloud InfluxDB data source
- Flux VS code extensions
- InfluxDB OSS 2.x dashboards